aspnetcore/blazor/hosting-models.md

```diff
@@ -48,6 +48,121 @@ There are downsides to Blazor WebAssembly hosting:
 * Download size is larger, and apps take longer to load.
 * .NET runtime and tooling support is less mature. For example, limitations exist in [.NET Standard](/dotnet/standard/net-standard) support and debugging.
 
+Blazor WebAssembly supports the [.NET SignalR client](xref:signalr/dotnet-client). To use SignalR in a Blazor WebAssembly app:
```

Comment:
I would prefer for this content to get integrated into the getting started tutorial for SignalR. Currently, that tutorial shows how to use the Signalr JS client. We could add tabs to that tutorial that show you how to use the .NET client from the browser instead if you are running a Blazor WebAssembly app. @bradygaster What do you think?

Reply:
Actually, it looks like we already have separate tutorials for JS & TS, so we could just add another one for .NET with Blazor.

Reply:
https://docs.microsoft.com/en-us/aspnet/core/tutorials/signalr

Reply:
I'll get this resolved.

Reply:
@guardrex Until we get agreement with the SignalR team I think we should pull this content out of this PR and treat it separately.

Reply:
Ok ... I'll open an issue then on hold.

Reply:
I'm going to hold it on this current issue that I opened for what to do with config content until we have a home for it ...

https://github.com/aspnet/AspNetCore.Docs/issues/16764#issuecomment-579916238